### PR TITLE
Add more Microsoft domains to MDFP list

### PR DESCRIPTION
**Pull request by ghostwords**
To fix various `*.live.com` problems. Sample reports:

>url: https://outlook.live.com/mail/...
>message: When viewing an attachment in outlook.com, the extension blocks attachment.outlook.office.net, so PDF attachments never load.

>url: https://office.live.com/start/Word.aspx?...
>block: s1-odc-15.cdn.office.net
>message: It starts blocking the CDN's

>url: https://office.live.com/start/Excel.aspx?...
>block: omextemplates.content.office.net,s1-excel-15.cdn.office.net
>message: This app blocked Office 365 Word, PowerPoint & Excel. The page only loads when I slide it to green from red.

All added domains should belong to the same entity (Microsoft) and were seen as blocked on domains belonging to the same entity. I haven't confirmed the breakage or this fix, however.

### CODE REVIEW

File: src/js/multiDomainFirstParties.js [hunk 1]
```diff
@@ -253,10 +253,12 @@ var multiDomainFirstPartiesArray = [
   ["mi.com", "xiaomi.com"],
   [
     "microsoft.com",
+
```
Comment (andresbase) [inline src/js/multiDomainFirstParties.js]: is this space on purpose?

Comment (ghostwords) [reply src/js/multiDomainFirstParties.js]: It is, it's the most-recognizable name, the only one that's not alphabetized. Just trying to organize things for readability, I dunno.

Comment (andresbase) [reply src/js/multiDomainFirstParties.js]: Oh that makes sense.

**Comment by ghostwords** [comment]
Here are the domains reported blocked on various `*.dynamics.com` pages:
```
+------------------------------+-------+
| blocked_fqdn                 | count |
+------------------------------+-------+
| assets.onestore.ms           |     3 |
| mem.gfx.ms                   |     3 |
| c.bing.com                   |     2 |
| m.webtrends.com              |     2 |
| platform.linkedin.com        |     2 |
| app.clickdimensions.com      |     1 |
| c.s-microsoft.com            |     1 |
| clientlog.portal.office.com  |     1 |
| dc.ads.linkedin.com          |     1 |
| js-agent.newrelic.com        |     1 |
| lptag.liveperson.net         |     1 |
| mscom.demdex.net             |     1 |
| mscrmapp.clickdimensions.com |     1 |
| notifications.google.com     |     1 |
| r1.res.office365.com         |     1 |
+------------------------------+-------+
```

**Comment by ghostwords** [comment]
Error report counts by page domain and exact blocked subdomain for `office.net`:
```
+---------------------------------------+--------------------------------------+-------+
| fqdn                                  | blocked_fqdn                         | count |
+---------------------------------------+--------------------------------------+-------+
| office.live.com                       | s1-odc-15.cdn.office.net             |    21 |
| outlook.live.com                      | attachment.outlook.office.net        |    11 |
| onedrive.live.com                     | s1-officeapps-15.cdn.office.net      |    10 |
| support.office.com                    | support.content.office.net           |     8 |
| office.live.com                       | omextemplates.content.office.net     |     7 |
| outlook.live.com                      | s1-officeapps-15.cdn.office.net      |     7 |
| outlook.live.com                      | s1-word-view-15.cdn.office.net       |     7 |
| onedrive.live.com                     | s1-word-edit-15.cdn.office.net       |     6 |
| onedrive.live.com                     | s1-onenote-15.cdn.office.net         |     4 |
| support.office.com                    | videoplayercdn.osi.office.net        |     4 |
| templates.office.com                  | omextemplates.content.office.net     |     3 |
| office.live.com                       | s1-excel-15.cdn.office.net           |     3 |
| onedrive.live.com                     | s1-excel-15.cdn.office.net           |     3 |
| onedrive.live.com                     | s1-word-view-15.cdn.office.net       |     3 |
| onedrive.live.com                     | contentstorage.osi.office.net        |     2 |
| stores.office.com                     | eus-contentstorage.osi.office.net    |     2 |
| [redacted].sharepoint.com             | s1-officeapps-15.cdn.office.net      |     2 |
...
```

File: src/js/multiDomainFirstParties.js [hunk 2]
```diff
@@ -266,12 +268,18 @@ var multiDomainFirstPartiesArray = [
     "msocdn.com",
     "office365.com",
     "office.com",
+    "office.net",
+    "onenote.com",
+    "onenote.net",
```
Comment (ghostwords) [inline src/js/multiDomainFirstParties.js]: Now I don't see anything for `onenote.net`, not sure why I added it.

**Comment by ghostwords** [comment]
Here are the domains reported blocked on `www.onenote.com` pages:
```
+---------------------------------+-------+
| blocked_fqdn                    | count |
+---------------------------------+-------+
| c.bing.com                      |    15 |
| onenote.officeapps.live.com     |    12 |
| nexus.ensighten.com             |     5 |
| googleads.g.doubleclick.net     |     3 |
| odc.officeapps.live.com         |     3 |
| static.doubleclick.net          |     3 |
| pipe.skype.com                  |     2 |
| dc.services.visualstudio.com    |     1 |
| docusign2com-my.sharepoint.com  |     1 |
| login.live.com                  |     1 |
| s1-officeapps-15.cdn.office.net |     1 |
| s1-onenote-15.cdn.office.net    |     1 |
+---------------------------------+-------+
```

**Comment by ghostwords** [comment]
Error report counts by page domain and exact blocked subdomain for `skypeassets.com`:
```
+------------------------+-----------------------------------+-------+
| fqdn                   | blocked_fqdn                      | count |
+------------------------+-----------------------------------+-------+
| www.skype.com          | secure.skypeassets.com            |    47 |
| login.skype.com        | apps.skypeassets.com              |    37 |
| secure.skype.com       | secure.skypeassets.com            |    23 |
| secure.skype.com       | apps.skypeassets.com              |    22 |
| lw.skype.com           | apps.skypeassets.com              |    13 |
| login.skype.com        | secure.skypeassets.com            |    12 |
| www.skype.com          | www.skypeassets.com               |    10 |
| support.skype.com      | secure.skypeassets.com            |     9 |
| web.skype.com          | static-asm.secure.skypeassets.com |     9 |
| outlook.live.com       | static-asm.secure.skypeassets.com |     6 |
| web.skype.com          | apps.skypeassets.com              |     3 |
| support.microsoft.com  | secure.skypeassets.com            |     2 |
| www.msn.com            | www.skypeassets.com               |     2 |
| outlook.live.com       | apps.skypeassets.com              |     1 |
| join.skype.com         | join.secure.skypeassets.com       |     1 |
| community.skype.com    | secure.skypeassets.com            |     1 |
| join.skype.com         | secure.skypeassets.com            |     1 |
| lw.skype.com           | secure.skypeassets.com            |     1 |
| web.skype.com          | secure.skypeassets.com            |     1 |
| www.msn.com            | secure.skypeassets.com            |     1 |
| [rdctd].mail.live.com  | static-asm.secure.skypeassets.com |     1 |
| [rdctd].mail.live.com  | static-asm.secure.skypeassets.com |     1 |
| [rdctd].mail.live.com  | static-asm.secure.skypeassets.com |     1 |
| [rdctd].mail.live.com  | static-asm.secure.skypeassets.com |     1 |
| [rdctd].mail.live.com  | static-asm.secure.skypeassets.com |     1 |
| onedrive.live.com      | static-asm.secure.skypeassets.com |     1 |
+------------------------+-----------------------------------+-------+
```

**Comment by ghostwords** [comment]
Here are the domains reported blocked on `www.yammer.com` pages:
```
+------------------------------------+-------+
| blocked_fqdn                       | count |
+------------------------------------+-------+
| c.bing.com                         |     9 |
| mug0.assets-yammer.com             |     9 |
| portal.office.com                  |     8 |
| word-view.officeapps.live.com      |     7 |
| r1.res.office365.com               |     6 |
| s0.assets-yammer.com               |     6 |
| clientlog.portal.office.com        |     5 |
| outlook.office365.com              |     3 |
| usc-word-view.officeapps.live.com  |     2 |
| word-edit.officeapps.live.com      |     2 |
| excel.officeapps.live.com          |     1 |
| login.live.com                     |     1 |
| prod.msocdn.com                    |     1 |
| usc-powerpoint.officeapps.live.com |     1 |
+------------------------------------+-------+
```

**Comment by ghostwords** [comment]
Here are the domains reported blocked on various `*.xbox.com` pages:
```
+------------------------+---------------------------------+-------+
| fqdn                   | blocked_fqdn                    | count |
+------------------------+---------------------------------+-------+
| www.xbox.com           | ad.atdmt.com                    |    56 |
| www.xbox.com           | assets.onestore.ms              |    55 |
| www.xbox.com           | mem.gfx.ms                      |    51 |
| www.xbox.com           | nexus.ensighten.com             |    45 |
| www.xbox.com           | c.bing.com                      |    38 |
| www.xbox.com           | tags.bluekai.com                |    21 |
| support.xbox.com       | api.facebook.com                |    19 |
| account.xbox.com       | ad.atdmt.com                    |    15 |
| support.xbox.com       | assets.onestore.ms              |    15 |
| account.xbox.com       | mem.gfx.ms                      |    15 |
| support.xbox.com       | mem.gfx.ms                      |    14 |
| support.xbox.com       | ad.atdmt.com                    |    13 |
| account.xbox.com       | assets.onestore.ms              |    13 |
| account.xbox.com       | c.bing.com                      |    13 |
| www.xbox.com           | images-eds-ssl.xboxlive.com     |    12 |
| support.xbox.com       | c.bing.com                      |    10 |
| www.xbox.com           | dpm.demdex.net                  |    10 |
| account.xbox.com       | login.live.com                  |    10 |
| store.xbox.com         | achievements.xboxlive.com       |     8 |
| marketplace.xbox.com   | assets.onestore.ms              |     8 |
| www.xbox.com           | fast.mscom.demdex.net           |     8 |
| www.xbox.com           | i.s-microsoft.com               |     8 |
| marketplace.xbox.com   | mem.gfx.ms                      |     8 |
...
```

**Comment by ghostwords** [comment]
Here are the domains reported blocked on various `*.visualstudio.com` pages:
```
+-------------------------------------+-----------------------------+-------+
| fqdn                                | blocked_fqdn                | count |
+-------------------------------------+-----------------------------+-------+
| www.visualstudio.com                | mem.gfx.ms                  |    63 |
| www.visualstudio.com                | assets.onestore.ms          |    59 |
| www.visualstudio.com                | js-agent.newrelic.com       |    41 |
| www.visualstudio.com                | nexus.ensighten.com         |    35 |
| www.visualstudio.com                | www.googletagmanager.com    |    20 |
| www.visualstudio.com                | i2-vso.sec.s-msft.com       |    17 |
| www.visualstudio.com                | i-vso.sec.s-msft.com        |    16 |
| www.visualstudio.com                | i3-vso.sec.s-msft.com       |    15 |
| www.visualstudio.com                | bam.nr-data.net             |    14 |
| www.visualstudio.com                | www.google-analytics.com    |    14 |
| www.visualstudio.com                | login.live.com              |    12 |
| developercommunity.visualstudio.com | assets.onestore.ms          |     9 |
| developercommunity.visualstudio.com | mem.gfx.ms                  |     9 |
...
```

**Comment by ghostwords** [comment]
Both `assets.onestore.ms` and `mem.gfx.ms` were reported blocked on `www.healthvault.com`.
>Missing icons. Can't login

>css is not applied

>display of all graphics on this site were eliminated.

**Pull request merged**